Makefile

```diff
@@ -23,16 +23,19 @@ run: all
 	$(H)cd www && python -m SimpleHTTPServer 8000
 
 check-deps: \
-	which-opam which-lua5.1 opam-config which-markdown yamlpp
+	which-opam which-markdown \
+	opkg-opam-file-format opkg-ocamlfind \
+	yamlpp scripts/archive2web.native
```

Comment:
here you check for `archive2web.native` but the `all:` target up there calls just `archive2web`

Reply:
also, given that we use `dune exec` in CI I guess we should use it also in `all:`

Reply:
finally, given you use nix to express requirements, you should probably remove this check in the first place